StRoot/StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -2502,6 +2536,181 @@ void StPicoDstMaker::fillMtdHits() {
   } //while (triggerPos.size() > 0)
 }
 
+//_________________
+void StPicoDstMaker::fillFwdTracks() {
+
+  StEvent *evt = (StEvent *)GetDataSet("StEvent");
+  if ( evt ){
+    StFwdTrackCollection * evc = evt->fwdTrackCollection();
+    if ( !evc ){
+      LOG_ERROR << "null FwdTrackCollection" << endm;
+      return;
+    }
```

Comment:
Here again we have the sub-optimal logic, as `fillFwdTracks()` probably should only be called when the StFwdTrackCollection exists, and it isn't necessarily a problem if that doesn't exist.
```suggestion
    if ( !evc ) return;
```